home/tests/tests.py

```diff
@@ -55,3 +57,35 @@ def test_do_not_change_queryset_when_parent_page_is_not_a_section(self):
         pages_after = limit_page_chooser(pages_before, request)
 
         self.assertEqual(pages_after, pages_before)
+
+
+class SurveyTranslationTest(TestCase):
```

Comment:
My intention behind this was to check all fields for at least one language in the future (to keep this PR small and to the point). Doing that will ensure that these kinds of bugs don't go overlooked.

Reply:
I wonder whether this can be done more efficiently with a static code analysis tool, or perhaps there is a single test or script that could be run to test all fields are translatable. My preference would be to scrap this test.